tasks/ca/statcan/census2016/census_columns_filter.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Nope, just a good text editor and some editing tricks to avoid dying in the process. I thought about creating some tooling but as it would be only valid for Canada 2016 (only one shot), I preferred not to do it